### ISSUE TITLE
Launch Chatty without GUI

### ISSUE THREAD

**Issue by lucarza** [chatty/chatty#452]
Hi everyone! First of all thanks for the awesome work you're doing with this!

**Request**

I'd like to gather some data logs for a NLP study l'm doing, so I decided to connect simultaneously to specific streamers for a long time just to get chat data from each. However, I've seen the CPU usage of Chatty increasing over time until it stops working.

I don't see any command line launch option to prevent the GUI from appearing, maybe it could be a useful feature for me or other "chats analysts" like me using Chatty, as I think it should considerably reduce the CPU usage

Or any other hints to reduce CPU usage is highly appreciated!

**Comment by tduva** [comment]
I don't have any plans to make Chatty work without a GUI, since it isn't really the goal of the program.

If you leave Chatty running for a very long time you may be running out of memory. You can [increase the max memory](https://chatty.github.io/help/help-memory_usage.html#restrict) Chatty can use, but there are also some settings that may help (under `Main - Settings`):

* Badges - Untick "Show Badges"
* Emoticons - Untick "Show emoticons"
* Chat - Reduce "Default chat buffer size"
* Moderation - Message History - Reduce "Limit per-user message history to"
* Live Streams - Untick "Request followed streams"

**Comment by arrowgent** [comment]
yes leaving chatty running for extended periods can use 12-15GB of ram
there are some things you can do to mitigate this

**Comment by mrandersonsmith** [comment]
you can just download logs post stream https://github.com/dfoverdx/twitch-chat-log-downloader
